aptos-move/move-examples/my_first_dapp/client/src/App.tsx

```diff
@@ -34,9 +34,9 @@ function App() {
   const fetchList = async () => {
     if (!account) return [];
     try {
-      const todoListResource = await provider.getAccountResource(
-        account?.address,
-        `${moduleAddress}::todolist::TodoList`,
+      const todoListResource = await aptos.getAccountResource(
+        {accountAddress:account?.address,
+          resourceType:`${moduleAddress}::todolist::TodoList`}
```

Comment:
We can actually make this nicer if we create a type also for this one like `getTableItem`